epsagon/trace.py

```diff
@@ -887,6 +902,27 @@ def remove_ignored_keys(self, input_dict):
                     copied_dict[key] = self.remove_ignored_keys(value)
         return copied_dict
 
+    def get_dict_with_allow_keys(self, input_dict):
```

Comment:
maybe just get_allowed_keys?

Reply:
but it's a dictionary that contains allowed keys, and not just keys.